server/app/services/geo/ServiceAreaInclusionGroup.java

```diff
@@ -0,0 +1,40 @@
+package services.geo;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Contains methods for serializing and deserializing stored values for {@link
+ * ServiceAreaInclusion}.
+ */
+public final class ServiceAreaInclusionGroup {
+  private ServiceAreaInclusionGroup() {}
+
+  public static ImmutableList<ServiceAreaInclusion> deserialize(String serviceAreas) {
+    ImmutableList.Builder<ServiceAreaInclusion> listBuilder = ImmutableList.builder();
+    for (String serviceAreaString : Splitter.on(",").split(serviceAreas)) {
+      List<String> serviceAreaParts = Splitter.on("_").splitToList(serviceAreaString);
+      listBuilder.add(
+          ServiceAreaInclusion.builder()
+              .setServiceAreaId(serviceAreaParts.get(0))
+              .setState(ServiceAreaState.getEnumFromSerializedFormat(serviceAreaParts.get(1)))
+              .setTimeStamp(Long.parseLong(serviceAreaParts.get(2)))
```

Comment:
**suggestion (blocking):** handle the exception for when there aren't values where they're expected and log the corrupted string for easier debugging

Reply:
Added exception handling.